src/main/java/org/folio/des/service/util/JobMapperUtil.java

```diff
@@ -0,0 +1,102 @@
+package org.folio.des.service.util;
+
+import org.apache.commons.lang3.ObjectUtils;
+import org.folio.de.entity.Job;
+import org.folio.de.entity.bursarlegacy.LegacyJob;
+import org.folio.des.domain.dto.ExportType;
+import org.folio.des.domain.dto.Metadata;
+
+public class JobMapperUtil {
+
+  private JobMapperUtil() {
+    throw new IllegalStateException("Utility class");
+  }
+
+  public static org.folio.des.domain.dto.Job entityToDto(Job jobEntity) {
+    var result = new org.folio.des.domain.dto.Job();
+
+    result.setId(jobEntity.getId());
+    result.setName(jobEntity.getName());
+    result.setDescription(jobEntity.getDescription());
+    result.setSource(jobEntity.getSource());
+    result.setIsSystemSource(jobEntity.getIsSystemSource());
+    result.setType(jobEntity.getType());
+    result.setExportTypeSpecificParameters(
+      jobEntity.getExportTypeSpecificParameters()
+    );
+    result.setStatus(jobEntity.getStatus());
+    if (
```

Comment:
please use the same style code convention, statement should be on the same line after opening brackets